### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by artur-abliazimov**

### CODE REVIEW

File: integration-tests/tests/soroban.converter.ts [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+import { bufToBigint } from 'bigint-conversion';
+
+type ElementType<T> = T extends Array<infer U> ? U : never;
+type KeyType<T> = T extends Map<infer K, any> ? K : never;
+type ValueType<T> = T extends Map<any, infer V> ? V : never;
+
+export function scvalToBigInt(scval: xdr.ScVal | undefined): BigInt {
+    switch (scval?.switch()) {
+        case undefined: {
+            return BigInt(0);
```
Comment (rudewalt) [inline integration-tests/tests/soroban.converter.ts:12]: It's better to return `undefined`

File: integration-tests/tests/soroban.converter.ts [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+}
+
+export function scValToJs<T>(val: xdr.ScVal): T {
+    switch (val?.switch()) {
+        case xdr.ScValType.scvBool(): {
+            return val.b() as unknown as T;
+        }
+        case xdr.ScValType.scvVoid():
+        case undefined: {
+            return 0 as unknown as T;
```
Comment (rudewalt) [inline integration-tests/tests/soroban.converter.ts:52]: It's better to return `undefined`

File: integration-tests/tests/soroban.client.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            allowHttp: true
+        });
+        this.client.getHealth();
+    }
+
+    async sendTransaction(
+        contractId: string,
+        method: string,
+        signer: string,
+        secret: string,
```
Comment (rudewalt) [inline integration-tests/tests/soroban.client.ts:18]: It is more convenient to wrap signer and secret into keypair struct or use some exists

File: integration-tests/tests/soroban.client.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            await delay(1000);
+            result = await this.client.getTransaction(response.hash);
+            attempts--;
+        } while (result.status === "NOT_FOUND" && attempts > 0);
+
+        return result;
+    }
+}
+
+async function delay(ms: number): Promise<void> {
```
Comment (mn13) [inline integration-tests/tests/soroban.client.ts:51]: I believe that `util.promisify` is more conventional for this

**Pull request merged**